src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/SecurityFactoryInterface.php

```diff
@@ -31,7 +46,18 @@ public function create(ContainerBuilder $container, $id, $config, $userProvider,
      */
     public function getPosition();
 
+    /**
+     * Defines the configuration key used to reference the provider
+     * in the firewall configuration.
+     * 
+     * @return string
+     */
     public function getKey();
 
+    /**
+     * Adds custom security configuration options.
+     * 
+     * @param NodeDefinition $builder
+     */
```

Comment:
I don't think this comment adds any new information about the method or its parameters.